static/styles/styles.less

```diff
@@ -172,69 +171,117 @@ Desktop:
 }
 
 .codepen {
-    border: solid 1px #dfdfdf;
-    display: block;
-    position: relative;
-    text-align: center;
-    top: -1.1rem;
-    border-top: none;
-    background-color: #dfdfdf;
-    color: gray;
-    padding: 3px;
-    cursor: pointer;
-    line-height: 18px;
+  border: solid 1px #dfdfdf;
+  display: block;
+  position: relative;
+  text-align: center;
+  top: -1.1rem;
+  border-top: none;
+  background-color: #dfdfdf;
+  color: gray;
+  padding: 3px;
+  cursor: pointer;
+  line-height: 18px;
 }
 article .codepen:before {
-    content: "Run in your browser";
+  content: "Run in your browser";
 }
 
 html {
-    color: #404040;
-    font: normal normal normal 1rem/1.6 -apple-system,BlinkMacSystemFont,Roboto,Helvetica Neue,Helvetica,Arial,sans-serif;
-    -webkit-font-smoothing: antialiased;
+  color: #404040;
+  font: normal normal normal 1rem/1.6 Helvetica, Arial, sans-serif;
+  -webkit-font-smoothing: antialiased;
 }
 
 h2 {
-    margin-top: 4em;
```

Comment:
The giant top margin was nice for participants and sharing on a screen.  It sort of "clears" the rest of the page.